vfox-restServer/src/main/java/com/vfoexchange/restServer/model/Services.java

```diff
@@ -0,0 +1,51 @@
+package com.vfoexchange.restServer.model;
+
```

Comment:
Remove unnecessary blank line.

Reply:
Unnecessary blank line removed and committed.